exporters/trace/zipkin/src/main/java/io/opencensus/exporter/trace/zipkin/ZipkinExporterHandler.java

```diff
@@ -53,8 +54,12 @@
 final class ZipkinExporterHandler extends TimeLimitedHandler {
   private static final Logger logger = Logger.getLogger(ZipkinExporterHandler.class.getName());
   private static final String EXPORT_SPAN_NAME = "SendZipkinSpans";
-  private static final String STATUS_CODE = "census.status_code";
-  private static final String STATUS_DESCRIPTION = "census.status_description";
+
+  // The naming follows Zipkin convention. As an example see:
+  // https://github.com/apache/incubator-zipkin-brave/blob/643b7245c462dc14d47afcdb076b2603fd421497/instrumentation/grpc/src/main/java/brave/grpc/GrpcParser.java#L67-L73
+  @VisibleForTesting static final String STATUS_CODE = "census.status_code";
```

Comment:
one important thing is that we do use non-ok as "error" message. this controls analysis etc, so I would recommend including this logic here.

Reply:
Got it, added in https://github.com/census-instrumentation/opencensus-java/pull/1931/commits/c1f588a81d5c9bb94b981969a72adb77a550a63e.